### PR TITLE
Update summary from string to array of strings

### PR DESCRIPTION
**Pull request by armoj**
update the summary shape to cater for multiple declarations

dependant on https://github.com/CriminalInjuriesCompensationAuthority/q-schema/pull/14 being approved and merged

proposed release version: `v4.0.0`

### CODE REVIEW

**Review by neil-stephen-mcgonigle — APPROVED** [review]
Happy with proposed release version: v4.0.0

**Review by neil-stephen-mcgonigle — APPROVED** [review]
I keep forgetting to check out and run the tests for these branches as there are no automated tests! Looks good and all tests passing.

File: index.js [hunk 1]
```diff
@@ -142,7 +142,13 @@ function createQuestionnaireTemplateHelper({
 
     // 2.3 - Does "summary" have a corresponding route
     function ensureSummaryRouteExists() {
-        return routeExists(routes.summary, '/routes/summary');
+        const results = routes.summary.map(x => routeExists(x, '/routes/summary'));
+        if (results.every(x => x === true)) {
```
Comment (sinclairs) [inline index.js:146]: Can you name the `x` variables appropriately?

**Review by sinclairs — CHANGES_REQUESTED** [review]

**Review by sinclairs — APPROVED** [review]
👍

**Pull request merged**